src/components/calcite-stepper-item/calcite-stepper-item.scss

```diff
@@ -142,30 +134,29 @@
   margin-right: initial;
 }
 :host([disabled]) {
-  opacity: var(--calcite-ui-opacity-disabled);
+  @apply opacity-disabled;
 }
 :host([disabled]),
 :host([disabled]) * {
-  cursor: not-allowed;
-  pointer-events: none;
+  @apply cursor-not-allowed pointer-events-auto;
 }
 
 :host([complete]) {
   // todo dark theme
   border-top-color: rgba($h-bb-060, 0.5);
   & .stepper-item-icon {
-    color: var(--calcite-ui-brand);
+    color: theme("backgroundColor.brand");
   }
 }
 
 :host([error]) {
-  border-top-color: var(--calcite-ui-danger);
+  @apply border-t-color-danger;
   & .stepper-item-number {
-    color: var(--calcite-ui-danger);
+    color: theme("backgroundColor.danger");
```

Comment:
Should we add these status colors to the `textColor` config?

Reply:
I think so, there are a few places we use them

Reply:
added to list